diff --git a/node_modules/pm2/lib/God/ActionMethods.js b/node_modules/pm2/lib/God/ActionMethods.js
index e8dc68b..ffe7740 100644
--- a/node_modules/pm2/lib/God/ActionMethods.js
+++ b/node_modules/pm2/lib/God/ActionMethods.js
@@ -57,7 +57,7 @@ module.exports = function(God) {
       }))
     }
 
-    pidusage(pids, function retPidUsage(err, statistics) {
+    pidusage(pids, {usePs: true}, function retPidUsage(err, statistics) {
       // Just log, we'll set empty statistics
       if (err) {
         console.error('Error caught while calling pidusage');